pkg/util/clusteroperator/status.go

```diff
@@ -0,0 +1,65 @@
+package clusteroperator
```

Comment:
you can use `https://github.com/openshift/library-go/tree/master/pkg/config/clusteroperator/v1helpers`